### PR TITLE
K jay/kinmen/status api

### PR DESCRIPTION
**Pull request by KJay221**
+ connect new API
+ add choose page button

### CODE REVIEW

File: src/components/status/index.tsx [hunk 1]
```diff
@@ -55,35 +56,51 @@ interface IStatus {
...
 const Status: React.FC = () => {
+  const user = JSON.parse(
+    localStorage.getItem('BEMS_USER') ||
+      sessionStorage.getItem('BEMS_USER') ||
+      '{}',
+  );
+  // data
   const [listInfo, setListInfo] = useState<IListInfo[]>([]);
   const [DRResult, setDRResult] = useState<IDR[]>([]);
+  const [DRPage, setDRPage] = useState<IDRPage[]>([]);
```
Comment (coliver0221) [inline src/components/status/index.tsx]: I think this parameter's name is not clear.

File: src/components/status/list/content.tsx [hunk 2]
```diff
@@ -98,6 +100,14 @@ const Content: React.FC<IContent> = ({
     'status-list-content-bidTypeNeed',
     'status-list-content-bidType',
   );
+  const bidTypeNow = classnames(
+    'status-list-content-bidTypeNow',
+    'status-list-content-bidType',
+  );
+  const bidTypeYesterday = classnames(
```
Comment (coliver0221) [inline src/components/status/list/content.tsx]: I think it's better to use `DayAhead` instead of `Yesterday`.

**Review by coliver0221 — CHANGES_REQUESTED** [review]

File: src/components/status/list/content.tsx (hunk 2 above)
Comment (KJay221) [reply src/components/status/list/content.tsx]: done

File: src/components/status/index.tsx (hunk 1 above)
Comment (KJay221) [reply src/components/status/index.tsx]: done

**Review by coliver0221 — COMMENTED** [review]
LGTM, thanks.

**Review by coliver0221 — APPROVED** [review]

**Pull request merged**